test/fixtures/dialects/mysql/create_database.sql

```diff
@@ -0,0 +1,18 @@
+CREATE DATABASE my_database;
+
+CREATE DATABASE IF NOT EXISTS my_database;
+
+CREATE DATABASE my_database
+DEFAULT CHARACTER SET utf8mb4
+COLLATE utf8mb4_0900_ai_ci
+DEFAULT ENCRYPTION 'N';
+
+CREATE DATABASE my_database
+DEFAULT CHARACTER SET = utf8mb4
+COLLATE = utf8mb4_0900_ai_ci
+DEFAULT ENCRYPTION = 'N';
+
+CREATE SCHEMA my_database
```

Comment:
Can you separate this out into a different file? It's not the same statement type

Reply:
On MySQL, CREATE SCHEMA is a synonym for CREATE DATABASE, as described in https://dev.mysql.com/doc/refman/8.0/en/create-database.html

So, I think it's better to combine statements than separate tests. What do you think?
However, I don't know how to disable ansi.CreateSchemaStatementSegment.
If you know how to do it, please let me know.

Reply:
> However, I don't know how to disable ansi.CreateSchemaStatementSegment.
> If you know how to do it, please let me know.

I found 'remove' argument of copy() is for that. I'll try.